benchexec/tablegenerator/react-table/src/utils/filters.js

```diff
@@ -165,15 +165,15 @@ const buildMatcher = (filters) => {
       acc[tool] = {};
     }
     let filter;
-    if (value.includes(":")) {
+    if (type === "measure" && value.includes(":")) {
```

Comment:
I just noticed that this misses `count` columns. In `utils.js` we have `isNumericColumn`, which unfortunately is not directly applicable here, but this could be refactored to reuse the code there?

Reply:
I've fixed this by using the `isNumericColumn` function and passing the `type` variable as an object. Might still not be the best solution, since we don't have a real column object here, but I think it's still better than just copying the code. What do you think?